src/actions/query.js

```diff
@@ -253,6 +253,7 @@ function msearch(
 									component,
 									response.hits,
 									response.took,
+									response.hits.hidden,
```

Comment:
@ShahAnuj2610 It'll break when `hits` is not defined, consider the error response.

Reply:
@bietkul done